DungeonSource/WarriorAttackBehavior.java

```diff
@@ -5,7 +5,7 @@
     @Override
     public void attack(DungeonCharacter hero, String displayName, DungeonCharacter opponent)
     {
-		System.out.println(displayName + " casts a spell of fireball at " +
+		System.out.println(displayName + " swings their sword at " +
```

Comment:
I also fixed the warrior attack string output so it no longer says "casts a fireball". Sorry, no magic warriors here.